src/nucleus/tt.ml

```diff
@@ -135,10 +133,12 @@ and instantiate es ?(lvl=0) ({term=e';assumptions;loc;} as e) =
   in
   match e' with
 
-    | Type -> {e with assumptions}
+    | Type -> {e with assumptions} (* XXX why do we need assumptions here? Type is well formed by itself *)
```

Comment:
Ah, ok, maybe we should put in a comment for the next time I look at this and wonder the same thing.